docker/build_image.sh

```diff
@@ -96,7 +99,10 @@ do
         # CUDA 10 is not supported on Ubuntu 20.04
         -ub|--ubuntu)
```

Comment:
We should kill this flag unless we're going to explicitly test and support Ubuntu 22.04

Reply:
Removed it